test/integration/good/vec-expr/pretty.expected

```diff
@@ -115,57 +116,67 @@ generated quantities {
 }
 parameters {
   real<lower=0, upper=1> theta;
 }
 transformed parameters {
   real y = 12;
-  row_vector[100] d = [1.0, 2.0, 3.0, y, y, y, 7, 8, 9.0, 10, 1.0, 2.0, 3.0,
-                       y, y, y, 7, 8, 9.0, 10, 1.0, 2.0, 3.0, y, y, y, 7, 8,
-                       9.0, 10, 1.0, 2.0, 3.0, y, y, y, 7, 8, 9.0, 10, 1.0,
-                       2.0, 3.0, y, y, y, 7, 8, 9.0, 10, 1.0, 2.0, 3.0, y, y,
-                       y, 7, 8, 9.0, 10, 1.0, 2.0, 3.0, y, y, y, 7, 8, 9.0,
-                       10, 1.0, 2.0, 3.0, y, y, y, 7, 8, 9.0, 10, 1.0, 2.0,
-                       3.0, y, y, y, 7, 8, 9.0, 10, 1.0, 2.0, 3.0, y, y, y,
-                       7, 8, 9.0, 10];
-  vector[100] e = [1.0, 2.0, 3.0, y, y, y, 7, 8, 9.0, 10, 1.0, 2.0, 3.0, y,
-                   y, y, 7, 8, 9.0, 10, 1.0, 2.0, 3.0, y, y, y, 7, 8, 9.0,
-                   10, 1.0, 2.0, 3.0, y, y, y, 7, 8, 9.0, 10, 1.0, 2.0, 3.0,
-                   y, y, y, 7, 8, 9.0, 10, 1.0, 2.0, 3.0, y, y, y, 7, 8, 9.0,
-                   10, 1.0, 2.0, 3.0, y, y, y, 7, 8, 9.0, 10, 1.0, 2.0, 3.0,
-                   y, y, y, 7, 8, 9.0, 10, 1.0, 2.0, 3.0, y, y, y, 7, 8, 9.0,
-                   10, 1.0, 2.0, 3.0, y, y, y, 7, 8, 9.0, 10]';
-  matrix[1, 100] f = [[1.0, 2.0, 3.0, y, y, y, 7, 8, 9.0, 10, 1.0, 2.0, 3.0,
-                       y, y, y, 7, 8, 9.0, 10, 1.0, 2.0, 3.0, y, y, y, 7, 8,
-                       9.0, 10, 1.0, 2.0, 3.0, y, y, y, 7, 8, 9.0, 10, 1.0,
-                       2.0, 3.0, y, y, y, 7, 8, 9.0, 10, 1.0, 2.0, 3.0, y, y,
-                       y, 7, 8, 9.0, 10, 1.0, 2.0, 3.0, y, y, y, 7, 8, 9.0,
-                       10, 1.0, 2.0, 3.0, y, y, y, 7, 8, 9.0, 10, 1.0, 2.0,
-                       3.0, y, y, y, 7, 8, 9.0, 10, 1.0, 2.0, 3.0, y, y, y,
-                       7, 8, 9.0, 10]];
+  row_vector[100] d = [1.0, 2.0, 3.0, y, y, y, 7, 8, 9.0, 10, 1.0, 2.0, 
+                       3.0, y, y, y, 7, 8, 9.0, 10, 1.0, 2.0, 3.0, y, 
+                       y, y, 7, 8, 9.0, 10, 1.0, 2.0, 3.0, y, y, y, 7, 
+                       8, 9.0, 10, 1.0, 2.0, 3.0, y, y, y, 7, 8, 9.0, 
+                       10, 1.0, 2.0, 3.0, y, y, y, 7, 8, 9.0, 10, 1.0, 
+                       2.0, 3.0, y, y, y, 7, 8, 9.0, 10, 1.0, 2.0, 3.0, 
+                       y, y, y, 7, 8, 9.0, 10, 1.0, 2.0, 3.0, y, y, y, 
+                       7, 8, 9.0, 10, 1.0, 2.0, 3.0, y, y, y, 7, 8, 9.0, 
+                       10];
+  
+  vector[100] e = [1.0, 2.0, 3.0, y, y, y, 7, 8, 9.0, 10, 1.0, 2.0, 3.0, 
+                   y, y, y, 7, 8, 9.0, 10, 1.0, 2.0, 3.0, y, y, y, 7, 
+                   8, 9.0, 10, 1.0, 2.0, 3.0, y, y, y, 7, 8, 9.0, 10, 
+                   1.0, 2.0, 3.0, y, y, y, 7, 8, 9.0, 10, 1.0, 2.0, 3.0, 
+                   y, y, y, 7, 8, 9.0, 10, 1.0, 2.0, 3.0, y, y, y, 7, 
+                   8, 9.0, 10, 1.0, 2.0, 3.0, y, y, y, 7, 8, 9.0, 10, 
+                   1.0, 2.0, 3.0, y, y, y, 7, 8, 9.0, 10, 1.0, 2.0, 3.0, 
+                   y, y, y, 7, 8, 9.0, 10]';
+  
```

Comment:
Is this additional space intentional?

Reply:
Yes. The empty line is also present in the original source file:
https://github.com/stan-dev/stanc3/blob/c620028502caadf14807be101341c1da54df6d1f/test/integration/good/vec-expr/vec_expr_big.stan#L66-L68
Most of these empty lines are from empty lines in the source but there's a couple of places where new unintentional lines appear. The problem is that when you open a local block with brace `{` the parser does not record the location of the brace but you'd usually put it on its own line like 
```stan
real x;
{
  real y = 2.0 + 2.0;
  x = y*y;
}
```
and now the pretty printer thinks there was an empty line...so it adds a superfluous line break before the block...

Reply:
Is this problem recurrent, or is it only on the first pass? 

Put another way, I have a problem `prog1.stan` which uses local blocks. If I pretty-print it to produce `prog2.stan`, it will have an additional space per block, understandable. If I now pretty-print `prog2.stan` to produce `prog3.stan`, with `prog3` be identical to `prog2`, or will it have even more whitespace?

Reply:
A sequence of multiple blank lines collapses to a single line. `prog3.stan` and `prog2.stan` should be identical.